src/BuildScriptGenerator/Python/PythonPlatform.cs

```diff
@@ -31,6 +31,12 @@ namespace Microsoft.Oryx.BuildScriptGenerator.Python
     [BuildProperty(
         TargetPackageDirectoryPropertyKey,
         "If provided, packages will be downloaded to the given directory instead of to a virtual environment.")]
+    [BuildProperty(
+        PythonPackageWheelPropertyKey,
+        "If provided, package wheels will be built with universal flag. For example," +
```

Comment:
nit: fix the content after "For example"

Reply:
Also do we need a 'value' for this property? since it is either universal wheel or not, we can just have "-p build_universa_wheel"?

Reply:
@kichalla ... imo value gives better flexibility .. in case python come up with some new  option 'foo' in future

Reply:
I had the same thought as @kichalla 

@arroyc -- in this case, I was thinking we could add an array of "supported package types", or something of the like, to this class, and reference that here in this description and in the validation checks later in the `GenerateBashBuildScriptSnippet` method.